### PR TITLE
Native bar chart styling

### PR DESCRIPTION
**Pull request by jordanch**
# todo

- [x] When the section number is correctly adjusted, [the widget's pull request](https://github.com/mendix/widgets-resources/pull/378) should be updated with the section number.
- [x] Merge only when widget is released.

### CODE REVIEW

File: content/refguide/native-styling-refguide.md [hunk 1]
```diff
@@ -1208,6 +1208,85 @@ A main object has four objects.
...
+<yAxis>
+	<label/>
+</yAxis>
+<legend>
+	<container/>
+	<item/>
+	<indicator/>
+	<label/>
+</legend>
+<lines>
```
Comment (IIsaku) [inline content/refguide/native-styling-refguide.md:1232]: I think you need to change this to "bars"

Comment (IIsaku) [reply content/refguide/native-styling-refguide.md:1232]: Same with the other xml elements below

Comment (jordanch) [reply content/refguide/native-styling-refguide.md:1232]: 👍

**Review by IIsaku — APPROVED** [review]

**Comment by jordanch** [comment]
This is ready to go :)

**Comment by ConnorLand** [comment]
This PR targeted a few docs which were affected by the Mx9 migration. Therefore Isa opened a new PR with the same changes: https://github.com/mendix/docs/pull/2987

It has been merged, so this PR can be closed.

**Pull request closed without merging**